packages/Tabs/stories/Tabs.stories.js

```diff
@@ -67,15 +76,77 @@ storiesOf(`${storyName}/Examples`, module)
       </Tabs>
     </Story>
   ))
-  .add("Compact Tabs with custom height", () => (
+  .add("Tabs with custom height", () => (
     <Story>
-      <Tabs>
-        <Tabs.List height={80}>
-          <Tabs.Tab hasInsetFocusStyle>These tabs</Tabs.Tab>
-          <Tabs.Tab hasInsetFocusStyle>Are tall</Tabs.Tab>
-          <Tabs.Tab hasInsetFocusStyle>And narrow</Tabs.Tab>
-          <Tabs.Tab hasInsetFocusStyle>With inset focus</Tabs.Tab>
+      <Tabs tabHeight={80} hasInsetFocusStyle>
```

Comment:
What is the argument for adding the new `hasInsetFocusStyle` prop? Shouldn't vertical tabs just always have inset focus style, and horizontal tabs dont even get the option?

Reply:
Sometimes they're needed for horizontal tabs too, if their container has `overflow:hidden`.  It wasn't explicitly part of the API before, you could add it to the `<Tab>` and since it was a `<RawButton>` that would get any `moreProps` it would work... but it wouldn't work when that `<Tab>` was a link (`<a>`).  

So this approach is a bit easier, you just need to add the prop once on the root `<Tabs>` and it gets propagated, and it works for links too.